rho/inventory_scan.py

```diff
@@ -313,9 +313,9 @@ def inventory_scan(hosts_yml_path, facts_to_collect, report_path,
                           forks=forks,
                           vars=json.dumps(ansible_vars))
 
-        rho_host_scan_timeout = os.getenv('RHO_HOST_SCAN_TIMEOUT', 10)
+        rho_host_scan_timeout = os.getenv('RHO_HOST_SCAN_TIMEOUT', 30)
```

Comment:
We should probably be checking that this value can be converted to an integer before casting it, and if not take the default value.
https://stackoverflow.com/a/1267145